isis/tests/PixelTests.cpp

```diff
@@ -465,7 +465,6 @@ TEST(Pixel, static_ToString) {
   EXPECT_EQ(std::string("Lis"), Pixel::ToString(Isis::Lis));
   EXPECT_EQ(std::string("Lrs"), Pixel::ToString(Isis::Lrs));
   EXPECT_EQ(std::string("Null"), Pixel::ToString(Isis::Null));
-  EXPECT_EQ(std::string("Invalid"), Pixel::ToString(-1.0e+1000));
```

Comment:
Is there any way we can test "Invalid" still without exceeding the valid range for doubles? Or is that what "Invalid" checks for?